pandora.py

```diff
@@ -11,6 +11,13 @@ class AudioFormat(enum.Enum):
     AACPLUS = 'aacplus'
 
 
+class TrackRating(enum.IntEnum):
```

Comment:
Same comment about this could be `enum.Enum` since its never used as an `int`

Reply:
On this one I do use the int.

See:
https://github.com/TingPing/pandora-rest/pull/14/commits/bb7189efc8fa6154b3b9f5e0b2d26d16806f6324#diff-c1ec1e0b2d287fbebda0b1d01535e36fR74

Reply:
A new song in a playlist will start as either 0 or 1 as a rating.

Reply:
Alright.